diff --git a/node_modules/howlongtobeat/dist/main/hltbsearch.js b/node_modules/howlongtobeat/dist/main/hltbsearch.js
index 16b1e47..2815483 100644
--- a/node_modules/howlongtobeat/dist/main/hltbsearch.js
+++ b/node_modules/howlongtobeat/dist/main/hltbsearch.js
@@ -106,7 +106,7 @@ class HltbSearch {
 }
 HltbSearch.BASE_URL = 'https://howlongtobeat.com/';
 HltbSearch.DETAIL_URL = `${HltbSearch.BASE_URL}game?id=`;
-HltbSearch.SEARCH_URL = `${HltbSearch.BASE_URL}api/search`;
+HltbSearch.SEARCH_URL = `${HltbSearch.BASE_URL}api/search/5683ebd079f1c360`;
 HltbSearch.IMAGE_URL = `${HltbSearch.BASE_URL}games/`;
 exports.HltbSearch = HltbSearch;
 //# sourceMappingURL=hltbsearch.js.map
\ No newline at end of file
diff --git a/node_modules/howlongtobeat/src/main/hltbsearch.ts b/node_modules/howlongtobeat/src/main/hltbsearch.ts
index a9bf05a..1c7a8f0 100644
--- a/node_modules/howlongtobeat/src/main/hltbsearch.ts
+++ b/node_modules/howlongtobeat/src/main/hltbsearch.ts
@@ -8,7 +8,7 @@ const UserAgent: any = require('user-agents');
 export class HltbSearch {
   public static BASE_URL: string = 'https://howlongtobeat.com/';
   public static DETAIL_URL: string = `${HltbSearch.BASE_URL}game?id=`;
-  public static SEARCH_URL: string = `${HltbSearch.BASE_URL}api/search`;
+  public static SEARCH_URL: string = `${HltbSearch.BASE_URL}api/search/5683ebd079f1c360`;
   public static IMAGE_URL: string = `${HltbSearch.BASE_URL}games/`;
 
   payload: any = {
